test/test_mps.py

```diff
@@ -8963,8 +8962,6 @@ class TestConsistency(TestCaseMPS):
     # by doing `EXPECTTEST_ACCEPT=1 python test_mps.py TestConsistencyCPU`
     # You most likely do NOT want to modify this manually
     ALLOWLIST_OP = {
-        'H': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
```

Comment:
I don't know why these tests are getting removed?

Reply:
not removed but it is duplicated in the list, and lint runner will complain same key different data.

Reply:
Got it.